src/default-directory-provider.coffee

```diff
@@ -26,7 +26,7 @@ class DefaultDirectoryProvider
 
     # TODO: Stop normalizing the path in pathwatcher's Directory.
     directory = new Directory(directoryPath)
-    if protocol?
+    if host
```

Comment:
Host is the new test. 

It's the same approach as used previously, only using a different part of the URL that is more reliably detected on Windows (but gives the same result as before on Mac, etc)